### ISSUE TITLE
Weird PCRE_ERROR_BADOPTION

### ISSUE THREAD

**Issue by ghost** [mscdex/node-pcre#6]
Setting these options PCRE.PCRE_MULTILINE | PCRE.PCRE_DOTALL causes execAll() not to work, but exec() works.

```
var PCRE = require('pcre').PCRE;

var content = '/**asd\nqwe\r\nzxc\rpoi*/'

var re = new PCRE( '/\\*\\*.*?\\*/', PCRE.PCRE_MULTILINE | PCRE.PCRE_DOTALL );
console.log( re.exec( content ) ); // works
console.log( re.execAll( content ) ); // PCRE_ERROR_BADOPTION

var re = new PCRE( '/\\*\\*.*?\\*/' );
console.log( re.execAll( content ) ); // works (no match)
```

**Comment by alexgorbatchev** [comment]
:+1:

**Comment by mscdex** [comment]
Should be fixed in 9b07495c6.
